src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -1741,8 +1755,12 @@ def create_function(cmd, resource_group_name, name, storage_account, plan=None,
     poller = client.web_apps.create_or_update(resource_group_name, name, functionapp_def)
     functionapp = LongRunningOperation(cmd.cli_ctx)(poller)
 
-    _set_remote_or_local_git(cmd, functionapp, resource_group_name, name, deployment_source_url,
-                             deployment_source_branch, deployment_local_git)
+    if(consumption_plan_location and is_linux):
+        logger.warning("Your Linux, cosumption plan, function app '%s' has been successfully created but is not active until content is published using"
```

Comment:
should we reword like "your linux function app of {}, which uses consumption plan, has been successfully, but it will not be activated till the content gets published"